extensions/interactions/customization-args-defs.ts

```diff
@@ -26,9 +26,7 @@ import { IReadableNote } from
   'extensions/interactions/MusicNotesInput/directives/oppia-interactive-music-notes-input.directive';
 
 
-export interface IAlgebraicExpressionInputCustomizationArgs {
-
-}
+export interface IAlgebraicExpressionInputCustomizationArgs { }
```

Comment:
Are we not removing the `I` prefix in this PR?

Reply:
lets do it in a separate pr. there has to be a other pr anyway.

Reply:
SGTM!